packages/gensx/src/checkpoint.ts

```diff
@@ -301,6 +314,24 @@ export class CheckpointManager implements CheckpointWriter {
           message: await response.text(),
         });
       }
+
+      if (this.printUrl && !this.havePrintedUrl) {
+        const responseBody = (await response.json()) as {
+          status: "ok";
+          data: {
+            executionId: string;
+            workflowName?: string;
+          };
+        };
+        const executionUrl = new URL(
+          `/${this.org}/workflows/${responseBody.data.workflowName ?? workflowName}/${responseBody.data.executionId}`,
+          this.consoleBaseUrl,
+        );
+        console.info(
+          `\n\n[GenSX] View execution at: ${executionUrl.toString()}\n\n`,
+        );
```

Comment:
Thoughts on colorizing this or making it a little "cuter"? Definitely want to get this merged, but just want to follow up on thoughts for making this feel "joyful" and having it pop a little bit cc @danfhernandez for ideas.